database.cc

```diff
@@ -1608,20 +1608,6 @@ future<> database::apply_hint(schema_ptr s, const frozen_mutation& m, tracing::t
     });
 }
 
-future<> database::apply_streaming_mutation(schema_ptr s, utils::UUID plan_id, const frozen_mutation& m, bool fragmented) {
```

Comment:
@avikivity I don't see you deleting the _streaming_dirty_memory_manager construction statement in the database constructor, or its .hh counterparts.

the streaming_concurrency_sem I guess is used by new repair too ?

There are also metrics in database.cc that reference the virtual dirty for streaming, but I guess those will naturally disappear once you remove the .hh declaration

Reply:
The semaphore is used by new streaming, including row-level repair.

Reply:
I planned to attack _streaming_dirty_memory_manager in a follow up, since it's more involved - we'll have to redistribute its 10%.

streaming_concurrency_sem  is still used.

Reply:
awesome.